Goobi/src/de/sub/goobi/forms/HelperForm.java

```diff
@@ -3,13 +3,6 @@
 /**
  * This file is part of the Goobi Application - a Workflow tool for the support of mass digitization.
  * 
- * Visit the websites for more information. 
- *     		- http://www.kitodo.org
- *     		- https://github.com/goobi/goobi-production
- * 		    - http://gdz.sub.uni-goettingen.de
- * 			- http://www.intranda.com
- * 			- http://digiverso.com 
- * 
  * This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free
```

Comment:
These lines contain several whitespace issues (tabs/blank, spaces at line endings) which are nagging when editing the file.

As the headers will need a major rework anyway, I started with removing unnecessary parts.